os/board/rtl8730e/src/component/wifi/inic/inic_ipc_host_trx.c

```diff
@@ -323,6 +329,11 @@ void inic_ipc_host_rx_handler(int idx_wlan, struct sk_buff *skb)
 	DCache_CleanInvalidate(((u32)skb - sizeof(struct list_head)), sizeof(struct skb_info));
 #endif /* CONFIG_ENABLE_CACHE */
 
+	/* free p_buf for the case: SUCCESS allocate pbuf but FAILED allocate precvbuf */
```

Comment:
The `RSP` tab is executed at both cases, success and failure.
When everything is ok (success case), this will free the `pbuf`. 
I think this is wrong. Could you confirm it again?

Reply:
Your statement is correct.
RSP label will go in as no return in between the function.
I have removed the comment.

Added the p_buf checking before freeing the p_buf due to line298: goto RSP 
If p_buf is NULL and goto RSP without checking the p_buf and do free will be an issue.